configs/espnetv1/README.md

```diff
@@ -0,0 +1,11 @@
+# ESPNet: Efficient Spatial Pyramid of Dilated Convolutions for Semantic Segmentation
+
+
+## Reference
+[1] Mehta S ,  Rastegari M ,  Caspi A , et al. ESPNet: Efficient Spatial Pyramid of Dilated Convolutions for Semantic Segmentation[J]. Springer, Cham, 2018.
+## Performance
+CityScapes
+
+| Model|steps|opt|image_size|batch_size|dataset|memory|card|mIou|config|
+| :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
+|ESPNet|120k|adam|1024x512|4|CityScapes|32G|4|0.6365|[espnet_cityscapes_1024x512_120k.yml](configs/espnetv1/espnetv1_cityscapes_1024x512_120k.yml)|
```

Comment:
Actually, you do not need to include so much information. Please refer to our style MD before you bring up your PR

Reply:
fixed